src/pages/landing/components/ConnectSection.tsx

```diff
@@ -16,6 +16,49 @@ declare global {
 
 const CardanoNS = window.cardano;
 
+const WalletSelection = (props: {
+  walletName: string|null;
+  activeWallets: string[];
+  loading: boolean;
+  handleSelectWallet: (walletName: string) => void;
+}) => {
+  const networkEnvVar = process.env.REACT_APP_WALLET_NETWORK;
+
+  if (props.loading) return <CircularProgress color="secondary" size={60} className="my-2" />;
+  if (props.walletName) return null;
+  if (!CardanoNS) return <Typography className="my-2 text-base">No wallet extensions installed yet!</Typography>;
+
+  return (
+    <Grid container spacing={2} justifyContent="center" alignItems="center">
+      {networkEnvVar !== '1' && ( 
```

Comment:
`networkEnvVar` should be 1 in .env
&
we show this warning when it is 1 - that is always. As right now in Cardano wallets, we cannot distinguish between _Preprod / Preview / Testnet_ (all with `networkId` 1). So, we leave it to the User to make sure he is connected to _Preprod_ by himself, or else his ADA might be lost in the network
@EehMauro

Reply:
My bad - :pray: 

**0** is testnet (testnet / preview / preprod) and **1** is mainnet